### PR TITLE
Add containerd and runc binaries to tgz output

### PR DESCRIPTION
**Pull request by crosbymichael**
This adds containerd and runc hashes as well as the binaries to the `tgz` make output.

### CODE REVIEW

File: hack/make.sh [hunk 1]
```diff
@@ -289,6 +289,28 @@ bundle() {
 	source "$SCRIPTDIR/make/$bundle" "$@"
 }
 
+copy_containerd() {
+    dir="$1"
+    # Add nested executables to bundle dir so we have complete set of
+    # them available, but only if the native OS/ARCH is the same as the
+    # OS/ARCH of the build target
+    if [ "$(go env GOOS)/$(go env GOARCH)" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then
```
Comment (kencochrane) [inline hack/make.sh]: I think you want.

```if [ "$GOOS/$GOARCH" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then``

What you had worked fine on ``cross`` but it wasn't working on ``tgz``

I ran it locally, and it seems to do what we want.

Comment (crosbymichael) [inline hack/make.sh]: fixed

Comment (kencochrane) [inline hack/make.sh]: cool, testing now

**Comment by kencochrane** [comment]
My tests passed :+1:

File: hack/make.sh (hunk 1 above)
Comment (kencochrane) [inline hack/make.sh]: tests look good.

**Comment by tiborvass** [comment]
LGTM

**Comment by tonistiigi** [comment]
LGTM

**Comment by calavera** [comment]
that `set -x` is quite noisy, but LGTM.

**Comment by crosbymichael** [comment]
@calavera that was there previously, i can remove it if you want

**Comment by tiborvass** [comment]
I like set -x, its the most important thing ever when doing a release

**Pull request merged**